src/components/SignUp/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from 'react';
+import { MainTitle, RegularBold, RegularText } from '@shared/Typography';
+import CustomButton from '@shared/Button';
+import CustomInput from '@shared/Input';
+import lock from '@assets/input/lock.svg';
+import mail from '@assets/input/mail.svg';
+import user from '@assets/input/user.svg';
+import {
+  AuthQuestions,
+  ButtonTitles,
+  Placeholders,
+  SubTitles,
+  Titles,
+} from '@enums/enums';
+import cl from './SignUp.module.scss';
+
+const GetStarted: FC = () => {
+  return (
+    <div className={cl.container}>
+      <MainTitle>{Titles.WelcometoStoryWire}</MainTitle>
+      <RegularText className={cl.textStyles} widthNormal={true}>
+        {SubTitles.WelcometoStoryWire}
+      </RegularText>
+      <div className={cl.box}>
+        <CustomInput text={Placeholders.Username} startInputIcon={user} />
```

Comment:
create a `helpers.ts` file here and array objects with input data and use the map to render inputs